Cargo.toml

```diff
@@ -21,9 +16,13 @@ unicode-width = "0.1.3"
 unicode-segmentation = "1.0"
 encode_unicode = "0.1.3"
 
-[target.'cfg(unix)'.dependencies]
+[target.'cfg(all(unix, not(any(target_os = "fuchsia"))))'.dependencies]
 nix = "0.8"
 
+[target.'cfg(target_os = "fuchsia")'.dependencies]
+fuchsia-zircon = "0.3.2"
+fuchsia-device = "0.1.0"
```

Comment:
This crate still needs to be published on crates.io

Reply:
Thanks you for this PR.
I will give your PR a try. I just need to figure how to install Fuchsia...
And I must warn you.
Currently, `rustyline` is stalled (but not dead).
But there are, at least, two alternatives: [liner](https://github.com/MovingtoMars/liner) and [linefeed](https://github.com/murarth/linefeed).
So maybe, you should consider these librairies as better alternatives...

Reply:
linefeed had too much GNU/Posix-isms in it for my taste. I'll check out liner though!

I'd recommend this resource: 
https://fuchsia.googlesource.com/docs/+/master/getting_started.md
If you have any problems trying Fuchsia, shoot me an email at bwb@google.com.

Reply:
Sorry, I gave up installing Fucshia: I have only an old MacBook Air...
Could please confirm that there is no pre-built vbox ?
Thanks.